### PR TITLE
feat: use `go-parse-duration` for time parameters

### LINKED ISSUE #68: feat: use `go-parse-duration`
## 🐞 Issue

Currently, commands with durational parameters such as /mute have way too many parameters just for the purpose of specifying the duration 
image

## ❗ Solution

Use a crate like `go-parse-duration` to take human speech/human readable time (such as 1d2h or 1 day 2 hours) and parse it into a duration

## ❓ Motivation

Reduce the number of necessary parameters just to specify time and clean the codebase from this 
image

### PR DESCRIPTION
**Pull request by Ushie**
Closes #68

### CODE REVIEW

File: src/commands/moderation.rs [hunk 1]
```diff
@@ -197,45 +198,26 @@ pub async fn unmute(
...
-    }
-    if let Some(days) = days {
-        mute_duration = mute_duration.checked_add(&Duration::days(days)).unwrap();
-    }
-    if let Some(months) = months {
-        const DAYS_IN_MONTH: i64 = 30;
-        mute_duration = mute_duration
-            .checked_add(&Duration::days(months * DAYS_IN_MONTH))
-            .unwrap();
-    }
```
Comment (oSumAtrIX) [inline src/commands/moderation.rs:238]: You can use match as an expression to set the variable duration.

File: src/utils/moderation.rs [hunk 2]
```diff
@@ -399,3 +400,8 @@ pub async fn mute_moderation(
 
     Ok((is_currently_muted, removed_roles))
 }
+
```
Comment (oSumAtrIX) [inline src/utils/moderation.rs:403]: I'd consider this a general utility function, not just moderation.

**Review by oSumAtrIX — APPROVED** [review]
Looks good

File: src/utils/moderation.rs (hunk 2 above)
Comment (Ushie) [reply src/utils/moderation.rs:403]: which file is appropiate for that? I see that the all the files in the utils dir are for specific scopes

Comment (oSumAtrIX) [reply src/utils/moderation.rs:403]: I think mod.rs inside utils shoud suffice

**Pull request merged**